documentation/modules/installing-openshift.adoc

```diff
@@ -0,0 +1,71 @@
+[[installing-openshift]]
+
+== Installing {ProductName} on OpenShift
+
+[[prerequisites]]
+=== Prerequisites
+
+To install {ProductName}, you need to have the OpenShift client tools. You can download OpenShift
+Origin client from https://github.com/openshift/origin/releases[OpenShift Origin]. {ProductName} is
+tested to work with the latest stable release.
+
+If you do not have an OpenShift cluster available, have a look at
+https://github.com/minishift/minishift[minishift] as a way of running a local instance of OpenShift
+on your machine.
+
+=== Installing
+
+Download one of the releases from
+https://github.com/EnMasseProject/enmasse/releases and unpack it. Once
+unpacked, you can deploy {ProductName} using the deployment script provided in the release.
+
+You can invoke the deployment script with `-h` to get a list of options.
+
+{ProductName} can be deployed in either a single-tenant mode or a multi-tenant mode. Choose
+single-tenant mode if you don't expect to be creating multiple isolated address spaces or if you do
+not have cluster-admin privileges. Choose multi-tenant mode otherwise.
+
+==== Installing in single-tenant mode
+
+[source,options="nowrap"]
+----
+./deploy-openshift.sh -m "https://localhost:8443" -n enmasse
+----
+
+This will create the deployments required for running {ProductName} in single-tenant mode. Starting
+up {ProductName} will take a while, usually depending on how fast it is able to download the docker
+images for the various components.
+
+You can use the REST API to check the status of the deployment.
```

Comment:
Make an ordered list item and add a plus sign above the code example below to include it as part of the item. Also change the period to a colon.